controllers/object_controls.go

```diff
@@ -1674,6 +1785,36 @@ func TransformMIGManager(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec,
 		setContainerEnv(&(obj.Spec.Template.Spec.Containers[0]), CDIEnabledEnvName, "true")
 	}
 
+	// Update the driver-root volume if a custom driver install directory is configured with the operator
+	if config.HostPaths.DriverInstallDir != "" && config.HostPaths.DriverInstallDir != DefaultDriverInstallDir {
+		for index, volume := range obj.Spec.Template.Spec.Volumes {
+			if volume.Name == "driver-root" {
+				obj.Spec.Template.Spec.Volumes[index].HostPath.Path = config.HostPaths.DriverInstallDir
+			}
+		}
+	}
+
+	// If a custom devPath is configured which is not equal to 'hostRoot/dev' or 'driverInstallDir/dev', make it available
+	// to the toolkit container under '/dev-root/dev'
+	devRoot := filepath.Dir(config.HostPaths.DevPath)
+	if config.HostPaths.DevPath != "" && devRoot != config.HostPaths.DriverInstallDir && devRoot != config.HostPaths.Root {
+		devVolume := corev1.Volume{
+			Name: "custom-dev",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: filepath.Join(devRoot, "/dev"),
+				},
+			},
+		}
+		obj.Spec.Template.Spec.Volumes = append(obj.Spec.Template.Spec.Volumes, devVolume)
+
+		devVolumeMount := corev1.VolumeMount{
+			Name:      "custom-dev",
+			MountPath: "/dev-root/dev",
+		}
+		obj.Spec.Template.Spec.Containers[0].VolumeMounts = append(obj.Spec.Template.Spec.Containers[0].VolumeMounts, devVolumeMount)
```

Comment:
same as previous

Reply:
I have removed devPath from the ClusterPolicy API now. Consequently, this code has been removed.